zigzag/zigzag_test_log.py

```diff
@@ -529,6 +542,17 @@ def _lookup_ids(self):
         except IndexError:  # test case has not been created yet in qTest
             pass
 
+    def _lookup_job_config_attributes(self):
+        """ Finds the array of job config attributes on the end of the job name.
+
+        Returns:
+            List: of job config attributes.
+        """
+        full_name = self._testcase_xml.items()[3][1]
```

Comment:
Can you address this by `self._testcase_xml.attrib['name']` the dict position is hard to understand and seems like it would be brittle.